repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/publication.ftl

```diff
@@ -1,9 +1,9 @@
 <#ftl output_format="HTML">
```

Comment:
If you are making changes to publication.ftl can you also update publicationPage.ftl as well please as the pageIndex rendering on both should be the same. (This was the reason for the pageIndex macro) Either update pageIndex macro or remove it and update both templates

Reply:
Cool

Reply:
@jacsamell I've updated both the `publication.ftl` and `publicationPage.ftl` files to implement the new sideNav and I got rid of the pageIndex partial, so all affected templates use the same mechanism to render the side navigation.